src/main.rs

```diff
@@ -18,7 +18,7 @@ impl Component for Model {
 
     fn create(_props: Self::Properties, link: ComponentLink<Self>) -> Self {
         Self {
-            input_value: "## Please Write Your MarkdEown!!".into(),
+            input_value: "## Please Write Your MarkDown!!".into(),
```

Comment:
Eが大文字だったのでMarkDownにしましたが、Markdownのほうが良いでしょうか？
```suggestion
            input_value: "## Please Write Your Markdown!!".into(),
```

Reply:
Markdown の方が正式名称ぽいのでこっちにします :bow:

https://en.wikipedia.org/wiki/Markdown